.openpublishing.redirection.json

```diff
@@ -4197,6 +4197,10 @@
     {
       "source_path": "docs/visual-basic/reference/vb6-support.md",
       "redirect_url": "https://docs.microsoft.com/previous-versions/visualstudio/visual-basic-6/visual-basic-6-support-policy"
+    },
+    {
+      "source_path": "docs/whats-new/2019-09.md",
```

Comment:
Is there a way to make this entry more generic? It would be nice if we didn't have to create another one of these entries each month.

Reply:
should you just let the link die then?

Reply:
I like just letting the link die. I'll make that change.

Reply:
RIP link.